dimensional.cabal

```diff
@@ -37,6 +37,11 @@ extra-source-files:  README.md,
                      examples/GM.lhs,
                      examples/NewtonianMechanics.hs
 
+flag binary
+  description: Provide instances for use with the binary package.
+  default:     False
```

Comment:
There was a discussion of whether the flag(s) should be true/false by default at https://github.com/bjornbm/dimensional/pull/111#issuecomment-173944523. It is fine to leave false if you prefer it that way (and less intrusive, I guess).

Reply:
Hmm this is a good question.
I'm really keen on keeping the flag off by default because we preservebackwards compatibility; no one will be surprised by a new dependency on binary, or a different version range constraint.

I'm happy to create another PR to change the flag to be on by default if people complain